instructor/models.py

```diff
@@ -187,17 +199,17 @@ class FlowCytometrySamplePrep(models.Model):
 
 class FlowCytometryHistogram(models.Model):
     sample_prep = models.ForeignKey(FlowCytometrySamplePrep, related_name='histograms')
-    strain_protocol = models.ForeignKey(StrainProtocol, related_name='histograms')
+    strain_protocol = models.ForeignKey(StrainTreatment, related_name='histograms')
     kind = models.CharField(max_length=50, choices=HISTOGRAMS, default=GAUSS)
     data = models.TextField(null=True,blank=True)
     enabled = models.BooleanField(default=False)
 
 admin.site.register(Course)
 admin.site.register(Assignment)
-admin.site.register(Protocol)
+admin.site.register(Drug)
 admin.site.register(Strains)
-admin.site.register(StrainProtocol)
-admin.site.register(Treatments)
+admin.site.register(StrainTreatment)
+admin.site.register(Treatment)
 admin.site.register(WesternBlot)
 admin.site.register(WesternBlotAntibody)
 admin.site.register(WesternBlotAntibodyBands)
```

Comment:
I am not an expert about schema migration, but it seems to me that this automatic generated migration is not complete. 
If you take a look at http://stackoverflow.com/a/12361162/257092 it seems you need to do something more than that (even if that migration is for renaming a table, not deleting it).
@ShawnMilo Do you have any suggestion?